everest-testing/src/everest/testing/ocpp_utils/fixtures.py

```diff
@@ -40,12 +40,21 @@ def ocpp_version(request) -> OCPPVersion:
 def ocpp_config(request, central_system: CentralSystem, test_config: OcppTestConfiguration, ocpp_version: OCPPVersion):
     ocpp_config_marker = request.node.get_closest_marker("ocpp_config")
 
+    ocpp_configuration_visitors_marker = request.node.get_closest_marker("ocpp_config_adaptions")
+    ocpp_configuration_visitors = []
+    if ocpp_configuration_visitors_marker:
+        for v in ocpp_configuration_visitors_marker.args:
+            assert hasattr(v,
+                           "adjust_ocpp_configuration"), "Arguments to 'ocpp_config_adaptions' must all provide interface of OCPPConfigAdjustmentVisitor"
+            ocpp_configuration_visitors.append(v)
```

Comment:
Suggestion: Break this part off into a separate fixture, like it was done for the EVerest config visitors

Reply:
I thought about this, but: This kind of could break things because adding this to `ocpp_utils/fixtures.py` would require an import in every usage of this fixture (and a "*" import from there is not recommended)
Also, here it is actually not necessary, because the `ocpp_config` fixture can be overwritten easily anyway (in contrast to the everest_core issue where overwriting the everest_core fixture would have been more elaborate and unstable since that is a volatile one)